WcaOnRails/app/webpacker/components/EditEvents/EventPanel/index.js

```diff
@@ -104,86 +104,66 @@ export default function EventPanel({
             : ''
         }
         onClick={() => dispatch(addEvent(wcifEvent.id))}
-        color="green"
-        size="tiny"
-        style={{
-          fontSize: '.75em',
-        }}
+        positive
+        size="small"
       >
         Add event
       </Button>
     );
   };
 
   return (
-    <Segment.Group
+    <Card
       size="tiny"
-      compact
       className={`event-panel event-${wcifEvent.id}`}
     >
-
-      <Header
-        className="event-panel__heading"
-        style={{
-          display: 'flex',
-          flexDirection: 'row',
-          alignItems: 'center',
-        }}
-        attached
+      <Card.Content
+        // replicate the way SemUI Cards handle images (borderless) without passing an actual image
+        style={{ padding: 0 }}
       >
-        <div
-          style={{
-            margin: '-1.5em -1em',
-          }}
-        >
-          <span className={cn('img-thumbnail', 'cubing-icon', `event-${event.id}`)} />
-        </div>
-        <span
-          className="title"
-          style={{
-            flexGrow: 1,
-            marginLeft: '2em',
-          }}
-        >
-          {event.name}
-
-        </span>
-        <div>
-          {renderRoundCountInputs()}
-        </div>
-      </Header>
-
+        <Segment basic tertiary>
+          <Header as="span">
+            <Icon className="cubing-icon" name={`event-${event.id}`} />
```

Comment:
I'm seeing errors in the console that the `name` prop provided is invalid. I think it's meant to be one of the thousands of pre-defined `SemanticICONS` options.

Reply:
Indeed this was because cubing icons are not predefined in Semantic.
I'm a bit surprised that there's no better support for custom icon sets, but anyways I've circumvented the error by use of the `cn` helper function